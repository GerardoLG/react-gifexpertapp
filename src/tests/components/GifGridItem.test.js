import { shallow } from "enzyme";
import GifGridItem from "../../components/GifGridItem";

describe("Tests on componetn <GifGridItem />", () => {
  test("should display the component correctly", () => {
    const wrapper = shallow(<GifGridItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
