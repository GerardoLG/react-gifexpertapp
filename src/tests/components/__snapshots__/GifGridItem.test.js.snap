// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on componetn <GifGridItem /> should display the component correctly 1`] = `
<div
  className="card animate__animated animate__delay-2s animate__fadeInDown"
>
  <img />
  <p />
</div>
`;
